TIM.ENH: remove unused channels from OrbIntlk.

ericonr opened the pull request:
Orbit interlock is enabled only for sectors 6, 8, 10 and 14, so the high level trigger doesn't need to control any other crate. This allows us to disable the trigger for other crates, to avoid spurious events causing a beam dump.

This only became an issue when we enabled event reception (RxEnbl) in the Fout devices for all timing AFCs. This was done in [1] with commit 656fb570 (orbintlk.ENH: fix Fout RxEnbl configuration to enable all outputs, 2025-04-09).

[1] https://github.com/lnls-sirius/dev-packages

anacso17 commented on timesys/high-level-triggers.py:
```diff
@@ -1812,104 +1833,44 @@
             },
         'channels': (
             'SI-01M1:DI-BPM:CRT4', 'SI-01M2:DI-BPM:CRT4',
-            'SI-01C1:DI-BPM-1:CRT4', 'SI-01C1:DI-BPM-2:CRT4',
```
why do we removed these BPMs? maybe we need to keep those that are related to BC

anacso17 replied:
I didn't understand the advantage of keeping all the other BPM CRT4 output without a known configuration (forcing them to be disabled, for example)

fernandohds564 replied:
Yeah, maybe it would be useful to have a high level trigger with the complement of these connections, to guarantee they are disabled, right?

anacso17 replied:
Yes, maybe the OutOrbIntlk high level trigger could be this one?

fernandohds564 replied:
Sounds good. However, in This case we need to be careful not to defining the same trigger output from the AMCFPGAEVR to both triggers. @ericonr, I think we could or come back the triggers from the same crate to this trigger here, or not include them in the new trigger, ok?

fernandohds564 replied:
hi @ericonr, I think I misunderstood the code. When I made the comment above I hadn't seen you had already created the other trigger I mentioned. I took a look again and it seems correct.

ericonr commented:
Added some additional explanations in commit messages and comments in the trigger definitions to warn that crates should only be included in one of the triggers. Hope this addresses all comments!
